src/components/Markdown.tsx

```diff
@@ -1,7 +1,5 @@
-import ReactMarkdown, {
-  PluggableList,
-  ReactMarkdownOptions,
-} from "react-markdown";
+import ReactMarkdown, { Options } from "react-markdown";
+import { PluggableList } from "unified";
```

Comment:
lets avoid using types directly from a transitive dependency. Can you update L11 to use `Options` from react-markdown instead?
`{ remark?: Options['remarkPlugins']; rehype: Options['rehypePlugins'] }`

Reply:
Fixed